benchmarks/500.scientific/5xx.channel_flow_jax_npbench/python/function.py

```
@@ -0,0 +1,279 @@
+                - 2 * ((u[2:, 0] - u[0:-2, 0]) / (2 * dy) * (v[1:-1, 1] - v[1:-1, -1]) / (2 * dx))
+                - ((v[2:, 0] - v[0:-2, 0]) / (2 * dy)) ** 2
+            )
+        )
+    )
+
+    return b
+
+
+@partial(jax.jit, static_argnums=(0,))
+def pressure_poisson_periodic(nit, p, dx, dy, b):
+    def body_func(p, q):
```

Ruff (0.14.8)

Unused function argument: `q`

(ARG001)